tests/beaker_tests/README-beaker-script-ref.md

```diff
@@ -0,0 +1,232 @@
+# Beaker Test Script Guide to Test Inputs
```

Comment:
'tests/beaker_tests/README-beaker-script-ref.md'

1) Does 'ref' mean 'reference', 'refactor', or what?
2) Why in tests/beaker_tests/ instead of in doc/? I'd prefer to keep all documentation in doc/.